### PR TITLE
Run code formatter on lib/elixir/lib/module.ex

### PR DESCRIPTION
**Pull request by udaysinghcode**

### CODE REVIEW

File: lib/elixir/lib/module.ex [hunk 1]
```diff
@@ -890,42 +905,58 @@ defmodule Module do
...
-            overridable_definitions = :elixir_overridable.overridable(module)
-            count = case :maps.find(tuple, overridable_definitions) do
-              {:ok, {count, _, _, _}} -> count + 1
-              :error -> 1
-            end
-            overridable_definitions = :maps.put(tuple, {count, clause, neighbours, false}, overridable_definitions)
-            :elixir_overridable.overridable(module, overridable_definitions)
-        end
+    :lists.foreach(
+      fn
```
Comment (josevalim) [inline lib/elixir/lib/module.ex]: Please extract the function  to a variable in the previous line and reformat the code.

File: lib/elixir/lib/module.ex [hunk 2]
```diff
@@ -890,42 +905,58 @@ defmodule Module do
...
+              :elixir_overridable.overridable(module, overridable_definitions)
+          end
 
-      other ->
-        raise ArgumentError,
-              "each element in tuple list has to be a {function_name :: atom, arity :: 0..255} tuple, got: #{inspect(other)}"
-    end, tuples)
+        other ->
+          raise ArgumentError,
+                "each element in tuple list has to be a {function_name :: atom, arity :: 0..255} tuple, got: #{
```
Comment (josevalim) [inline lib/elixir/lib/module.ex]: Please extract the error message to a variable.

File: lib/elixir/lib/module.ex [hunk 3]
```diff
@@ -890,42 +905,58 @@ defmodule Module do
...
   @spec make_overridable(module, module) :: :ok
   def make_overridable(module, behaviour) when is_atom(module) and is_atom(behaviour) do
     case check_module_for_overridable(module, behaviour) do
       :ok ->
         :ok
+
       {:error, error_explanation} ->
-        raise ArgumentError, "cannot pass module #{inspect(behaviour)} as argument to defoverridable/1 because #{error_explanation}"
+        raise ArgumentError,
+              "cannot pass module #{inspect(behaviour)} as argument to defoverridable/1 because #{
```
Comment (josevalim) [inline lib/elixir/lib/module.ex]: Please extract the error message to a variable.

File: lib/elixir/lib/module.ex [hunk 4]
```diff
@@ -960,11 +991,17 @@ defmodule Module do
...
+
       not function_exported?(behaviour, :behaviour_info, 1) ->
         {:error, "it does not define any callbacks"}
+
       behaviour not in behaviour_definitions ->
-        {:error, "its corresponding behaviour is missing. Did you forget to " <>
-                 "add @behaviour #{inspect(behaviour)}?"}
+        {
+          :error,
+          "its corresponding behaviour is missing. Did you forget to " <>
```
Comment (josevalim) [inline lib/elixir/lib/module.ex]: Please extract the error message to a variable.

File: lib/elixir/lib/module.ex [hunk 5]
```diff
@@ -1155,30 +1199,37 @@ defmodule Module do
...
     case add_doc(module, line, kind, pair, args, doc) do
       :ok ->
         :ok
+
       {:error, :private_doc} ->
-        :elixir_errors.warn line, env.file,
+        :elixir_errors.warn(
+          line,
+          env.file,
           "#{kind} #{name}/#{arity} is private, " <>
```
Comment (josevalim) [inline lib/elixir/lib/module.ex]: Please extract the error message to a variable.

File: lib/elixir/lib/module.ex [hunk 6]
```diff
@@ -1350,11 +1410,17 @@ defmodule Module do
...
         :ets.update_element(table, key, {4, nil})
         val
+
       [] when is_list(stack) ->
         # TODO: Consider raising instead of warning on v2.0 as it usually cascades
-        IO.warn "undefined module attribute @#{key}, " <>
-                "please remove access to @#{key} or explicitly set it before access", stack
+        IO.warn(
+          "undefined module attribute @#{key}, " <>
+            "please remove access to @#{key} or explicitly set it before access",
```
Comment (josevalim) [inline lib/elixir/lib/module.ex]: Please extract the error message to a variable.

File: lib/elixir/lib/module.ex [hunk 7]
```diff
@@ -1371,16 +1437,20 @@ defmodule Module do
     # TODO: Remove on Elixir v2.0
     case value do
       {:parse_transform, _} when key == :compile and is_list(stack) ->
-        IO.warn "@compile {:parse_transform, _} is deprecated. " <>
-                "Elixir will no longer support Erlang-based transforms in future versions", stack
+        IO.warn(
+          "@compile {:parse_transform, _} is deprecated. " <>
+            "Elixir will no longer support Erlang-based transforms in future versions",
```
Comment (josevalim) [inline lib/elixir/lib/module.ex]: Please extract the error message to a variable.

File: lib/elixir/lib/module.ex [hunk 8]
```diff
@@ -1399,40 +1469,50 @@ defmodule Module do
     case value do
       {line, doc} when is_integer(line) and (is_binary(doc) or is_boolean(doc) or is_nil(doc)) ->
         value
+
       {line, doc} when is_integer(line) ->
         raise ArgumentError,
-              "expected the #{key} attribute to contain a binary, a boolean, or nil, got: #{inspect(doc)}"
+              "expected the #{key} attribute to contain a binary, a boolean, or nil, got: #{
```
Comment (josevalim) [inline lib/elixir/lib/module.ex]: Please extract the error message to a variable.

File: lib/elixir/lib/module.ex [hunk 9]
```diff
@@ -1399,40 +1469,50 @@ defmodule Module do
...
+
+      module when is_atom(module) and module != nil ->
         # Attempt to compile behaviour but ignore failure (will warn later)
         _ = Code.ensure_compiled(module)
         value
+
       other ->
         raise ArgumentError,
-              "expected the @impl attribute to contain a module or a boolean, got: #{inspect(other)}"
+              "expected the @impl attribute to contain a module or a boolean, got: #{
```
Comment (josevalim) [inline lib/elixir/lib/module.ex]: Please extract the error message to a variable.

File: lib/elixir/lib/module.ex (hunk 1 above)
Comment (udaysinghcode) [reply lib/elixir/lib/module.ex]: How exactly should I go about this? Should I 

`def func do 
{...}
end
`
within the make_overridable function scope, or outside its scope?

Or simply move the function a line higher, give it a name, and then stick it into :lists.foreach/2?

Comment (josevalim) [reply lib/elixir/lib/module.ex]: > Or simply move the function a line higher, give it a name, and then stick it into :lists.foreach/2?

This. Just assign it to a variable right before.

**Comment by josevalim** [comment]
Ping/

**Comment by udaysinghcode** [comment]
Hey @josevalim - busy day at work. Let me just clean this up right now, just getting home.

**Comment by josevalim** [comment]
@udaysinghcode no worries, I just wanted to make sure you didn't miss it. :) I will merge this in 8 hours, so take your time.

**Pull request merged**